shoulder_pan_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: shoulder_pan_joint
    joint_speed: 2
    motor:
        id: 11
        init: 666
        min: 212
        max: 812

elbow_tilt_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: shoulder_pan_joint
    joint_speed: 2
    motor:
        id: 9
        init: 512
        min: 212
        max: 812

wrist_tilt_controller:
    controller:
        package: ax12_controller_core
        module: joint_position_controller
        type: JointPositionControllerAX12
    joint_name: wrist_rotate_joint
    joint_speed: 2
    motor:
        id: 10
        init: 512
        min: 212
        max: 822

